.dockerignore

```diff
@@ -39,18 +39,30 @@
 !target/x86_64-unknown-linux-gnu/release/agent
 !target/x86_64-unknown-linux-gnu/debug/udev-video-broker
 !target/x86_64-unknown-linux-gnu/release/udev-video-broker
+!target/x86_64-unknown-linux-gnu/release/udev-discovery-handler
```

Comment:
the other binaries have exceptions for release and debug (debug isn't really used anywhere, so it is fine not to add it)